src/Abp.EntityFramework/EntityFramework/AbpDbContext.cs

```diff
@@ -339,18 +340,40 @@ protected virtual void CheckAndSetId(object entityAsObj)
             }
         }
 
-        EdmProperty GetEdmProperty(Type type, string propertyName)
+        EdmProperty GetEdmProperty(Type type)
         {
-            var metadata = ((IObjectContextAdapter)this).ObjectContext.MetadataWorkspace;
+            var metadata = ((IObjectContextAdapter) this).ObjectContext.MetadataWorkspace;
```

Comment:
Don't remove space after type cast parentheses.

Reply:
@acjh did you mean, don't add space after type cast parentheses ?

Reply:
Whoops yes.

Reply:
I have modified the code like that 😄